lib/api/workloadendpoint.go

```diff
@@ -29,7 +31,8 @@ type WorkloadEndpoint struct {
 type WorkloadEndpointMetadata struct {
 	unversioned.ObjectMetadata
 
-	// The name of the endpoint.
+	// The name of the endpoint.  This may be omitted on a create, in which case an endpoint
+	// ID will be automatically created, and the endpoint ID will be included in the response.
```

Comment:
That is my plan.  I have a separate "renaming task".  I realize I could have just done that here too, but half of this code already existed so I was planning on doing that alongside the hostname->node and pool->ipPool stuff.  OK if I leave it out of this one?